x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/authc/AuthenticationServiceTests.java

```diff
@@ -688,6 +688,7 @@ public void testRealmLookupThrowingExceptionRest() throws Exception {
         }
     }
 
+    @SuppressWarnings("unchecked")
```

Comment:
Why is this suppression still necessary (and the others in this file)?

Reply:
In this particular case L699 `i.getArguments()` returns an Object[] which needs to be cast to ActionListener<User>. Some of the Mockito stuff in here also requires SupressWarnings but I haven't added it elsewhere to keep this PR focused in raw tapes removals. I also haven't removed some other warnings in the classes touched since I think its makes any clean-up PR like this harder to review.